editor/src/java/com/dynamo/upnp/Request.java

```diff
@@ -64,7 +65,7 @@ public static Request parse(String header) {
             } else {
                 int index = token.indexOf(":");
                 if (index != -1) {
-                    String key = token.substring(0, index).toUpperCase();
+                    String key = token.substring(0, index).toUpperCase(Locale.ROOT);
```

Comment:
This and the other upnp classes below have also received the same fix. Not sure if it ever is a problem here though. Should I revert this?

Reply:
Changed on global level in Bob